Code/Framework/AzFramework/AzFramework/Viewport/ViewportBus.h

```diff
@@ -77,4 +77,17 @@ namespace AzFramework
     };
 
     using ViewportBorderRequestBus = AZ::EBus<ViewportBorderRequests>;
+
+    class ViewportImGuiNotifications : public AZ::EBusTraits
+    {
+    public:
+        /*!
+         * Notification that the ImGui menu bar has been activated or deactivated.
+         *
+         * \param active True/False when ImGui is activated/deactivated.
+         */
+        virtual void ImGuiActive(const bool /*active*/){};
+    };
+
+    using ViewportImGuiNotificationsBus = AZ::EBus<ViewportImGuiNotifications>;
```

Comment:
update: Please rename this to `ViewportImGuiNotificationBus` (drop the `s` from `Notifications`, naming reads better without the plural there - we try and do this for all buses that follow this pattern)